bwapi/BWAPI/Source/BWAPI/AutoMenuManager.cpp

```diff
@@ -11,6 +11,7 @@
 //#include <BW/Dialog.h>
 #include <BWAPI/GameImpl.h>
 //#include <BW/Offsets.h>
+#include <windows.h>
```

Comment:
windows.h is obviously a windows-only header, so we can't include it, at least not without it being inside a #ifdef _WIN32. Although in that case, the IsDebuggerPresent check should also be in a #ifdef _WIN32. 
Honestly the IsDebuggerPresent check should probably just be removed.

Reply:
If remove `IsDebuggerPresent` then probably autoMenuPause configuration should be completely removed. Also upon closed investigation whole `AutoMenuManager::onMenuFrame` is mystery for me.